### ISSUE TITLE
Android app

### ISSUE THREAD

**Issue by hananabilabd** [ruohoruotsi/Butterworth-Filter-Design#5]
Hi bro 
Um working on an android app to do DSP in real-time from mic to speaker , but um using a superpowered SDK and unfortunately it doesn't allow to increase the order of filter so can i import your library in my project and replace the old filter , so Can i use your library in real-time ?? and is there an example on how to use low-pass filter and band-pass ?
Thanks in advance

**Comment by ruohoruotsi** [comment]
Hi there, you can you use in realtime, I can provide a short example for how to use it in an online setup. Perhaps I'll add a unit-test that processes a WAV file in chunks, will that be good enough?

**Comment by hananabilabd** [comment]
yeah sure , all i wanna do is to pass an pointer of float array then receive a filtered array , that's all ,
